WalletWasabi.Fluent/Models/Wallets/WalletModel.cs

```diff
@@ -20,6 +20,7 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 public partial class WalletModel : ReactiveObject
 {
 	private readonly Lazy<IWalletCoinjoinModel> _coinjoin;
+	private readonly Lazy<IWalletCoinsModel> _coins;
```

Comment:
After this fix from @SuperJMN (https://github.com/zkSNACKs/WalletWasabi/pull/11548), you might not need the `Lazy` anymore, could you check it?

Reply:
> @soosr lazy is needed because the constructor of `WalletCoinsModel` depends on `WalletCoinjoinModel`, so it initializes the latter earlier and that results in a crash.
> 
> Maybe we should come up with some Rx-friendly way of waiting for required dependencies

I now wonder why is the crash happening. Is it due to sequential coupling or a circular dependency? Is it easy to explain? If it's not, we can schedule a short call to understand the problem. This kind of design problems is worth discussing :)